charts/node-analyzer/values.yaml

```diff
@@ -264,7 +264,7 @@ nodeAnalyzer:
     probesPort: 7002
     image:
       repository: sysdig/vuln-runtime-scanner
-      tag: 1.4.12
+      tag: 1.5
```

Comment:
Not sure if this may be treated as a float by helm.

Reply:
I don't believe it will, but if you want to be sure quote it with `"`